Makefile

```diff
@@ -14,7 +14,7 @@ COMPILE_TARGET=./tmp/_output/bin/$(PROJECT)
 PROMETHEUS_OPERATOR_VERSION=v0.34.0
 LOCAL=local
 GRAFANA_OPERATOR_VERSION=v3.0.2
-AMO_VERSION=1.1.4
+AMO_VERSION=v1.1.4
```

Comment:
Why is the `v` needed here? and not needed in the `PREV_AMO_VERSION`?

Reply:
I think that can be removed also. In fact, I'm fairly sure that v breaks the gen commands.